invenio_drafts_resources/services/records/components/files.py

```diff
@@ -64,6 +67,14 @@ def update_draft(self, identity, data=None, record=None, errors=None):
                     ],
                 }
             )
+        elif draft.files.enabled and draft.files.items():
+            for item in draft.files.items():
+                data = item[1].file.dumps()
+                if not TransferType(data["storage_class"]).is_completed:
+                    raise ValidationError(
+                        _("Upload in progress. " "Please wait."),
+                        field_name="files.enabled",
```

Comment:
I would put it in the whole files field, I'm not sure if this would display correctly though
```suggestion
                        field_name="files",
```